src/core/pointcloud/qgseptpointcloudindex.cpp

```diff
@@ -446,6 +447,7 @@ bool QgsEptPointCloudIndex::loadHierarchy()
     if ( errH.error != QJsonParseError::NoError )
     {
       QgsDebugMsgLevel( QStringLiteral( "QJsonParseError when reading hierarchy from file %1" ).arg( filename ), 2 );
+      mError = QStringLiteral( "QJsonParseError when reading hierarchy from file %1" ).arg( filename );
```

Comment:
While this description is ok for the message log, it seems a bit too technical for the user.

Reply:
I prefer to have a too technical error message than saying that the point cloud just didn't load and not indicate what went wrong to the user + this message has the file name so it is easier to figure out what went wrong